Source/WTF/wtf/text/StringImpl.h

```diff
@@ -1354,6 +1358,62 @@ inline Ref<StringImpl> StringImpl::createByReplacingInCharacters(const UChar* ch
     return newImpl;
 }
 
+template<typename Func>
```

Comment:
Is there a reason we use `func` instead of `function`?

Reply:
Changed.